marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/default/main.sjs

```diff
@@ -94,7 +94,7 @@ function main(content, options) {
   }
 
   //now let's make our attachments, if it's xml, it'll be passed as string
-  instance['$attachments'] = doc;
+  instance['$attachments'] = instance;
```

Comment:
This seems odd - storing the instance itself as an attachment on the instance?

Reply:
That was the original code. I tested with it and got some error in cleanData of makeEnvelope. So I changed it back to doc. Maybe I missed a commit.

Reply:
The attachment instance here is the source document and not the harmonized instance. doc will give the entire envelope of the raw source data.

Reply:
So I did change the attachments part in the scaffolding. 
This part in the default-mapping/main.sjs gives me that STACKOVERFLOW error in cleanData(). Will push a commit changing it back to doc in the default-mapping/main.sjs